crates/solver/src/solver/http_solver.rs

```diff
@@ -467,7 +467,8 @@ impl Solver for HttpSolver {
                             external_prices.clone(),
                         )
                         .await?;
-                    tracing::debug!(
+                    // This can be a large log message so we don't want to log it by default.
+                    tracing::trace!(
```

Comment:
What does `by default` actually means? Do we need to turn this on somewhere to be logged or?
Asking because I used this log quite frequently recently.

Reply:
You can add to the `--log-filter` option `solver::solver::http_solver=trace` to print this log.